grumpy-runtime-src/testing/dict_test.py

```diff
@@ -31,7 +32,10 @@
 l = []
 for k in d:
   l.append(k)
-assert l == ['baz', 'foo', 'bar', 'qux']
+if hasattr(sys, "goversion"):
+  assert l == ['foo', 'bar', 'baz', 'qux']
+else:
+  assert l == ['baz', 'foo', 'bar', 'qux']
```

Comment:
This is testing that Grumpy have ordered dict or _not ordered_  dict?

Reply:
It checks that grumpy has order-preserving dictionary, and python2.7 has unordered dictionary.
Order for grumpy I wrote looking at assignments above.